supplementary_style_guide/glossary_terms_conventions/general_conventions/d.adoc

```diff
@@ -307,6 +307,17 @@ The API object for a deployment can be either a Kubernetes-native `Deployment` o
 
 *See also*:
 
+[discrete]
+[[developer-preview]]
+==== image:images/yes.png[yes] Developer Preview (noun)
+*Description*: _Developer Preview_ software provides early access to upcoming product software in advance of its possible inclusion in a Red Hat product offering. Customers can use Developer Preview software to test functionality and provide feedback during the development process. The software might not have any documentation, is subject to change or removal at any time, and has received limited testing. Developer Preview software is not supported by Red Hat in any way and is not functionally complete or production-ready.
```

Comment:
"early access to upcoming product software in advance of its possible inclusion in a Red Hat product offering" is redundant. Recommend deleting "in advance of its possible inclusion in a Red Hat product offering".

Reply:
This isn't redundant. See https://access.redhat.com/support/offerings/devpreview

"Developer Preview versions are provided to expose features from upstream communities allowing early adopters and integrators to explore and interact with new capabilities in advance of their _possible inclusion_ in a Red Hat product offering."